src/span.js

```diff
@@ -205,8 +205,10 @@ export default class Span {
    **/
   addTags(keyValuePairs: any): Span {
     if (opentracing.Tags.SAMPLING_PRIORITY in keyValuePairs) {
-      this._setSamplingPriority(keyValuePairs[opentracing.Tags.SAMPLING_PRIORITY]);
-      delete keyValuePairs[opentracing.Tags.SAMPLING_PRIORITY];
+      // Delete the SAMPLING_PRIORITY tag if it was not set
```

Comment:
If the span is already marked as debug (for instance from [`.isDebugIDContainerOnly`](https://github.com/jaegertracing/jaeger-client-node/blob/master/src/tracer.js#L217)) then the tag should be ignored so as to not hit the credits twice. Same for `. setTag(...)`, below.